Dockerfile

```diff
@@ -56,6 +56,25 @@ RUN mkdir -p /usr/src/app && \
 
 WORKDIR /usr/src/app
 
+# Install Chromium and Puppeteer for PDF generation
+# Installs latest Chromium package available on Alpine (Chromium 108)
+RUN apk add --no-cache \
+        chromium \
+        nss \
+        freetype \
+        harfbuzz \
+        ca-certificates \
+        ttf-freefont \
+        nodejs \
+        yarn
+
+# Tell Puppeteer to skip installing Chrome. We'll be using the installed package.
+ENV PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true
+ENV PUPPETEER_EXECUTABLE_PATH=/usr/bin/chromium-browser
+
+# Install latest version of Puppeteer that works with Chromium 108
+RUN yarn add puppeteer@19.2.0
```

Comment:
do we need to lock it to 19.2. [example from civl apply](https://github.com/ministryofjustice/laa-apply-for-legal-aid/blob/62f68bd05d19d90472885c6e63621cc50b18b246/docker/apply_base.dockerfile#L49)